pyuvdata/uvh5.py

```diff
@@ -102,7 +126,7 @@ def _read_header(self, header, filename):
         self.Nants_telescope = int(header['Nants_telescope'].value)
         self.ant_1_array = header['ant_1_array'].value
         self.ant_2_array = header['ant_2_array'].value
-        self.antenna_names = [uvutils._bytes_to_str(n) for n in header['antenna_names'].value]
+        self.antenna_names = [uvutils._bytes_to_str(n.tostring()) for n in header['antenna_names'].value]
```

Comment:
This line looks different from the others (it's not using `_read_uvh5_string`). Is that what you want?